plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -682,16 +682,18 @@ protected void renameTable(ConnectorSession session, String catalogName, String
     @Override
     public void finishInsertTable(ConnectorSession session, JdbcOutputTableHandle handle)
     {
-        RemoteTableName temporaryRemoteTableName = new RemoteTableName(
```

Comment:
Maybe we can `getTemporaryTable()` and `getTargetTable()` methods to `JdbcOutputTableHandle`?

Reply:
I think this could be a good idea, but it is more about refactoring and I don't want to overcomplicate this PR.